api/app/wildfire_one/wfwx_api.py

```diff
@@ -291,6 +291,15 @@ async def get_wfwx_stations_from_station_codes(
     return requested_stations
 
 
+async def get_wfwx_all_active_stations(
```

Comment:
You've removed the code that uses this - so I think we can drop this function now?